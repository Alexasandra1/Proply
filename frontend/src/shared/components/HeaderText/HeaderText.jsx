import React, { useState } from "react"

import './HeaderText.scss'

export function HeaderText(props) {
    function handleClick() {
        if (props.onClick) {
            props.onClick();
        }
    }

    return (
        <>
            <div className={`${props.style} ${props.dopStyle}`} 
            onClick={handleClick}
            >{props.word}</div>
        </>
    )
}