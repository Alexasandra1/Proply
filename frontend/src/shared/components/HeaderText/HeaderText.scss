.logIn {
  color: #2D7C83;
}

// .popupNameStyle{
//     color: #FFF;
//     text-align: center;
//     font-family: Montserrat;
//     font-size: 48px;
// }

// .popupStyle {
//     color: #FFF;
//     text-align: center;
//     font-family: Montserrat;
//     font-size: 32px;

// }

// .selectedStyle {
//     font-weight: 800;
//     color: #905050;
//     font-size: 2.2vh;
//     animation: textAnimation 0.5s;
// }


// @keyframes textAnimation {
//     0% {
//       font-size: 2vh;
//       color: #ffffff; 
//     }
//     100% {
//       font-size: 2.2vh;
//       color: #905050;
//     }
//   }


.footer__box__houses,
.footer__box__community,
.footer__box__aboutUs {
  color: #FFF;
  font-family: Montserrat;
  font-size: 20px;
}