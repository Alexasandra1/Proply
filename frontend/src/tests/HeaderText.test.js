import React from 'react';
import renderer from 'react-test-renderer';
import { HeaderText } from '../shared/components/HeaderText/HeaderText';

describe('HeaderText', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<HeaderText style="style" word="word" />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
