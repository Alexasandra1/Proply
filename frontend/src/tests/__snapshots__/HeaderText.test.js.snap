// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderText renders correctly 1`] = `
<div
  className="style undefined"
  onClick={[Function]}
>
  word
</div>
`;
